### PR TITLE
Release testplan: fix the option name for setting default version

### PR DESCRIPTION
**Pull request by clebergnu**
The location where one finds the option to set the new default to be
displayed by readthedocs.org is actually a different one.

Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

File: examples/testplans/release/release.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	 },
          {
 	 "name": "Configure Read The Docs",
-         "description": "Visit the link below:\n\nhttps://readthedocs.org/dashboard/avocado-framework/edit/\n\n 1) Click in *Versions*. In *Choose Active Versions*, find the version you're releasing and check the *Active* option. *Submit*;\n 2) Click in *Versions* again. In *Default Version*, select the new version you're releasing. *Submit*."
+         "description": "Visit the link below:\n\nhttps://readthedocs.org/dashboard/avocado-framework/edit/\n\n 1) Click in *Versions*. In *Choose Active Versions*, find the version you're releasing and check the *Active* option. *Submit*;\n 2) Click in *Admin*. In *Default Version*, select the new version you're releasing. *Submit*."
```
Comment (richtja) [inline examples/testplans/release/release.json:40]: I tried to follow the instructions and I think that still it's not correct. What about this?
1) Click in *Versions*. Under *Activate a version*, find the version you're releasing and check the *Activate* option. 
2) Click in *Admin*. Under *Advanced Settings* in *Default Version*, select the new version you're releasing. *Save*.

**Review by richtja — CHANGES_REQUESTED** [review]

File: examples/testplans/release/release.json (hunk 1 above)
Comment (clebergnu) [reply examples/testplans/release/release.json:40]: Yep, looks better.  Sending an update.

Comment (clebergnu) [reply examples/testplans/release/release.json:40]: I ended up with:

1) Click in *Versions*. Under *Activate a Version*, find the version you're releasing and click the *Activate* button;
2) Click in *Admin*. Under *Advanced Settings*/*Global settings*/*Default Version*, select the new version you're releasing. Click the *Save* button at the bottom of the page."

**Comment by willianrampazzo** [comment]
I cannot access `https://readthedocs.org/dashboard/avocado-framework/edit/`, getting 404, probably due to missing authorization. Could you add me to the `readthedocs` Avocado project?

**Comment by clebergnu** [comment]
> I cannot access `https://readthedocs.org/dashboard/avocado-framework/edit/`, getting 404, probably due to missing authorization. Could you add me to the `readthedocs` Avocado project?

Permission addded!

**Review by willianrampazzo — APPROVED** [review]

**Review by richtja — APPROVED** [review]
LGTM, thanks

**Review by beraldoleal — APPROVED** [review]
LGTM

**Pull request merged**